draft-ietf-webtrans-http2.md

```diff
@@ -85,38 +82,21 @@ draft corresponding to this document can be found at[]
 
 # Introduction
 
-This protocol satisfies the WebTransport protocol framework's requirements using
-HTTP semantics to provide support for unidirectional streams, bidirectional
-streams, and datagrams.
-
-By using HTTP semantics, this protocol allows deployment using any HTTP version,
-although this document in particular focuses on HTTP/2 as the current most
-common TCP-based fallback to HTTP/3 running over QUIC {{?RFC9000}}.
-
-Currently, the only mechanism in HTTP/2 for server to client communication is
-server push. That is, servers can initiate unidirectional push promised streams
-to clients, but clients cannot respond to them; they can only accept them or
-discard them. Additionally, intermediaries along the path may have different
-server push policies and may not forward push promised streams to the downstream
-client. This best effort mechanism is not sufficient to reliably deliver
-messages from servers to clients, limiting server to client use-cases such as
-chat messages or notifications.
-
-Several techniques have been developed to workaround these limitations: long
-polling {{?RFC6202}}, WebSocket {{?RFC8441}}, and tunneling using the CONNECT
-method. All of these approaches have limitations.
-
-This document defines a mechanism for establishing bidirectional communication
-using HTTP semantics in a manner that conforms with the WebTransport protocol
-requirements and semantics [OVERVIEW]. When running with HTTP/2, multiple
-WebTransport instances can be multiplexed simultaneously with regular HTTP
-traffic on the same HTTP/2 connection.
-
-It is important to note that, while it is possible to have the WebTransport
-session remain entirely "self contained" within a given HTTP flow, certain HTTP
-versions may provide native features, such as datagram support, which can allow
-for a better performing implementation of the WebTransport client's
-requirements.
```

Comment:
Do we want to talk about the ability to break out to native features at all? Not saying we need to, but that sort of feeds into the discussion about HTTP datagrams and such.

Reply:
I think that we can feed that into the wt-h3 draft.

I would like to see that use the design we have here, but then enrich it by allowing things to jump out into their own streams or datagrams.

Reply:
Makes sense. Putting some slides in for 112 about that since I expect we'll get a variety of thoughts from different folks on how this relates to H3.